packages/common/driver-definitions/package.json

```diff
@@ -107,15 +107,8 @@
 		"rimraf": "^4.4.0",
 		"typescript": "~5.4.5"
 	},
-	"fluidBuild": {
-		"tasks": {
-			"tsc": [
-				"typetests:gen"
-			]
-		}
-	},
```

Comment:
Why is this the best change? Only the test builds should depend on "typetest:gen" and packages that build tests in "tsc" and "build:esnext" should be the exceptions that have this called out. (This is a regression from my point of view in tracking where those aberrations live.)

Reply:
I removed these because they were wrong. They were saying "the _only_ thing that tsc depends on in this package is typetests:gen". I think what was _intended_ was to augment the global tsc definition, so it should be:

```json
	"fluidBuild": {
		"tasks": {
			"tsc": [
				"...",
				"typetests:gen"
			]
		}
	},
```

I'm testing those changes.